Traceback (most recent call last):
  File "/leonardo/home/userexternal/forazi00/Dim_red_swap_Leo/reconstruction_fidelity.py", line 157, in <module>
    main()
  File "/leonardo/home/userexternal/forazi00/Dim_red_swap_Leo/reconstruction_fidelity.py", line 50, in main
    for subfolder in os.listdir(mix_fold):
                     ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'out_mix'
